src/components/Sections/SectionList/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import './index.scss';
+
+const SectionList = ({
+  name,
+  description,
+  keywords,
+  url,
+  roles,
+  entity,
+  type,
+  startDate,
+  endDate,
+  highlights,
+  position,
+  summary,
+  organization,
+  sectionName,
+  reference,
+  language,
+  fluency,
+}) => {
+  const _highlights = highlights?.map((data, index) => {
+    return <li key={index} dangerouslySetInnerHTML={{ __html: data }}></li>;
+  });
+
+  const _keywords = keywords?.map((data, index) => {
+    return <li key={index} dangerouslySetInnerHTML={{ __html: data }}></li>;
+  });
+
+  const _roles = roles?.map((data, index) => {
+    return <li key={index} dangerouslySetInnerHTML={{ __html: data }}></li>;
+  });
```

Comment:
As you know react will not able to distinguish if given the same key index to every field. Better way of writing this would be like this

```suggestion
 const injectHTML = (content, key) => {
  return content?.map((data, index) => {
    return (
      <li
        key={`${key}_${index}`}
        dangerouslySetInnerHTML={{ __html: data }}
      ></li>
    );
  });
};

const _highlights = injectHTML(highlights, "_highlights");

const _keywords = injectHTML(keywords, "_keywords");

const _roles = injectHTML(roles, "_roles");
```